lib/sles4sap_publiccloud.pm

```diff
@@ -608,4 +610,48 @@ sub delete_network_peering {
     }
 }
 
+=head2 get_hana_database_info
+
+    Parses "hdbsql" command output.
+    Returns true if hana database is online.
+
+=cut
+
+sub get_hana_database_info {
+    my ($self) = @_;
+    my $result = -1;
+    my $output_cmd = $self->run_cmd(cmd => "hdbsql -u SYSTEM -p Linux1234 -i 00 'SELECT * FROM SYS.M_DATABASES;'", runas => get_required_var("SAP_SIDADM"));
+    my $hana_database_online = '"HDB","HDB-00","YES","ACTIVE","","","DEFAULT","?"';
+
+    $result = ($output_cmd =~ /$hana_database_online/ ? 1 : 0);
+    return $result;
+}
+
+=head2 is_hana_database_offline
+
+    $self->is_hana_database_offline();
+    is_hana_database_offline([timeout => 900]);
+    Check if hana DB is offline.
+
```

Comment:
if you like to describe the only argument `timeout` there's an example about how to do it in qesapdeploy.pm 
It is using `=over` , `=item` and `=back`

Reply:
Please mention also what is the `return` about

Reply:
refactored thx for the comment

Reply:
```suggestion
Returns 1 if hana database is online, 0 if offline
```